concepticondata/conceptlists/Paperno-2019-99.tsv

```diff
@@ -0,0 +1,100 @@
+Paperno-2019-99-5	5	belyj	white	color	1335	WHITE
+Paperno-2019-99-6	6	sinij	blue	color	837	BLUE
+Paperno-2019-99-7	7	želtyj	yellow	color	1424	YELLOW
+Paperno-2019-99-8	8	černyj	black	color	163	BLACK
+Paperno-2019-99-9	9	seryj	grey	color	153	GREY
+Paperno-2019-99-10	10	zelenyj	green	color	1425	GREEN
+Paperno-2019-99-11	11	krasnyj	red	color	156	RED
+Paperno-2019-99-12	12	temnyj	dark	color	706	DARK
+Paperno-2019-99-13	13	ravnyj	equal	comparison	1570	EQUAL
+Paperno-2019-99-14	14	raznyj	different	comparison	2504	DIFFERENT
+Paperno-2019-99-15	15	poxožij	similar	comparison	1558	SIMILAR
+Paperno-2019-99-16	16	podobnyj	analogous	comparison		
+Paperno-2019-99-17	17	levyj	left	direction	244	LEFT
+Paperno-2019-99-18	18	pravyj	right	direction	1019	RIGHT
+Paperno-2019-99-19	19	čužoj	foreign	emotional evaluation	109	FOREIGN
+Paperno-2019-99-20	20	krasivyj	beautiful	emotional evaluation	1427	BEAUTIFUL
+Paperno-2019-99-21	21	prekrasnyj	wonderful	emotional evaluation		
+Paperno-2019-99-22	22	dobryj 	kind	emotional evaluation	3299	KIND
+Paperno-2019-99-23	23	milyj	nice	emotional evaluation	3299	KIND
+Paperno-2019-99-24	24	rodnoj	native	emotional evaluation		
+Paperno-2019-99-25	25	dorogoj	dear	emotional evaluation	2470	DEAR
+Paperno-2019-99-26	26	nastojaščij	real	emotional evaluation		
+Paperno-2019-99-27	27	sčastlivyj	happy	emotional evaluation	1495	HAPPY
+Paperno-2019-99-28	28	xorošij	good	emotional evaluation	1035	GOOD
+Paperno-2019-99-29	29	nužnyj	necessary	emotional evaluation	1030	NECESSARY
+Paperno-2019-99-30	30	strašnyj	horrible	emotional evaluation		
+Paperno-2019-99-31	31	jasnyj	clear	emotional evaluation	989	CLEAR
+Paperno-2019-99-32	32	ploxoj	bad	emotional evaluation	1292	BAD
+Paperno-2019-99-33	33	blizkij	close	location properties	3717	NEAR (IN SPACE)
+Paperno-2019-99-34	34	dalekij	faraway	location properties	1406	FAR
+Paperno-2019-99-35	35	tixij	quiet	location properties	1314	QUIET
+Paperno-2019-99-36	36	poslednij	last	order	1529	LAST (FINAL)
+Paperno-2019-99-37	37	sledujuščij	next	order	1354	NEXT
+Paperno-2019-99-38	38	spokojnyj	calm	personality		
+Paperno-2019-99-39	39	uverennyj	confident	personality		
+Paperno-2019-99-40	40	veselyj	funny	personality	1976	MERRY
+Paperno-2019-99-41	41	bol'šoj	big	size	1202	BIG
+Paperno-2019-99-42	42	krupnyj	large	size	1202	BIG
+Paperno-2019-99-43	43	nebol'šoj	small	size	1246	SMALL
+Paperno-2019-99-44	44	ogromnyj	huge	size	1202	BIG
+Paperno-2019-99-45	45	malen'kij	little	size	1246	SMALL
+Paperno-2019-99-46	46	glubokij	deep	depth	1593	DEEP
+Paperno-2019-99-47	47	melkij	shallow	depth	193	SHALLOW
+Paperno-2019-99-48	48	nizkij	low	height	2191	LOW (HEIGHT)
+Paperno-2019-99-49	49	vysokij	high	height	1265	HIGH
+Paperno-2019-99-50	50	dlinnyj	long	length	1203	LONG
+Paperno-2019-99-51	51	korotkij	short	length	1645	SHORT
+Paperno-2019-99-52	52	širokij	wide	width	1243	WIDE
+Paperno-2019-99-53	53	tolstyj	thick	width	1244	THICK
+Paperno-2019-99-54	54	tonkij	thin	width	2249	THIN (OF SHAPE OF OBJECT)
```

Comment:
tonkij is used for both persons to indicate they are thin and for objects, so I would say: use the braoder THIN.